oat-sa/tao-test-runner-qti-fe
# Added a11y labelling items in the bottom bar

**Related to:**
https://oat-sa.atlassian.net/browse/ACTP-301

**Description:**
Fix the labelling of the menu tools in the bottom bar

**How to test**
1. Create a test with eneabled tools (like Calculator)
2. Start this test as a test tacker
3. Run screenreader
4. Navigate to the bottom bar the tools, menus and menu options should be readable
